### PR TITLE
Lvjav442 11

### PR DESCRIPTION
**Pull request by martavoloshyn**

### CODE REVIEW

File: src/main/java/com/softserve/ita/java442/cityDonut/model/ProjectStatus.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name="project_status")
+public class ProjectStatus {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @NotNull
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/ProjectStatus.java]: remove @ NotNull...add @Column(name = "status", nullable = false)...default length 255, you need length less than 255, and add length to column where it`s need

File: src/main/java/com/softserve/ita/java442/cityDonut/model/StoryBoard.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    @Column(name="description", length=1000)
+    private String description;
+
+    @NotNull
+    @Column(name = "date")
+    private LocalDateTime date;
+
+    @NotNull
+    @ManyToOne(cascade = {CascadeType.ALL}, fetch = FetchType.LAZY)
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/StoryBoard.java]: if we delete story board, project deletes

File: src/main/java/com/softserve/ita/java442/cityDonut/model/StoryBoard.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    @JoinColumn(name = "project_id")
+    private long projectId;
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/StoryBoard.java]: row 34 need to change; correct - private Project project

**Review by NazarRyvak — DISMISSED** [review]

File: src/main/java/com/softserve/ita/java442/cityDonut/model/StoryBoard.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    private String description;
+
+    @Column(name = "date", nullable = false)
+    private LocalDateTime date;
+
+    @ManyToOne(cascade = {
+            CascadeType.DETACH, CascadeType.MERGE,
+            CascadeType.PERSIST, CascadeType.REFRESH},fetch = FetchType.EAGER)
+    @JoinColumn(name = "project_id")
+    private Project project;
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/StoryBoard.java:33]: where is @OneToMany with field media?

**Review by NazarRyvak — DISMISSED** [review]

**Review by nastiamazurak — DISMISSED** [review]

**Review by nastiamazurak — APPROVED** [review]

**Review by NazarRyvak — APPROVED** [review]

**Pull request merged**